Scripts/Utilities/MaterialUtils.cs

```diff
@@ -105,6 +105,14 @@ public static Color RandomColor()
 			return new Color(r, g, b);
 		}
 
+		public static Color HueShift(Color color, float shift)
```

Comment:
Actually wouldn't that work? Since I'm setting .r, .g, etc. on the same color?

But yeah, simpler as a non-extension method.